code/src/App.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react'
+import { Provider } from 'react-redux'
+import { combineReducers, configureStore } from '@reduxjs/toolkit'
+import { BrowserRouter, Routes, Route, Navigate } from 'react-router-dom'
+
+import StartPage from 'components/StartPage'
+import IntroPage from 'components/Intropage'
+import tasks from 'reducers/tasks'
+import NotFound from 'components/NotFound'
+
+
+const reducer = combineReducers({
+  tasks: tasks.reducer,
+})
+
+const store = configureStore({reducer: reducer,})
```

Comment:
Just a small tip: You can write it like this if you want to:
const store = configureStore({ reducer })

Reply:
Thank you!